chatgpt/cli.py

```diff
@@ -26,14 +25,16 @@
 def setup():
     """Setup a chat."""
     console.print(f"Config directory: {CONFIG_DIR}")
-    if not os.path.exists(CONFIG_DIR.absolute()):
+    if not CONFIG_DIR.exists():
```

Comment:
I'm wondering... if later, there's a need to pass a path to the session key file, shouldn't be `CONFIG_DIR` removed? I mean the setup flow here looks like this:
- Check if `CONFIG_DIR` exists
- Create if it doesn't exist
- Ask for a path to session key file
- Read token from given path

`CONFIG_DIR` is not really used

Reply:
I removed the config directory... Because we need some location to store the key and logs. That would the .chatgpt-api in *home*. I also removed the confirmation to create it, because the directory would be created anyway in the logging part:
```python
if not LOGGING_DIR.exists():
    LOGGING_DIR.mkdir(parents=True, exist_ok=True)
```

There is still a warning that sates it was created.